client/me/purchases/add-credit-card/index.jsx

```diff
@@ -21,21 +21,25 @@ import Main from 'components/main';
 import titles from 'me/purchases/titles';
 import { billingHistory } from 'me/purchases/paths';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { withStripe } from 'lib/stripe';
+
+const CreditCardFormWithStripe = withStripe( CreditCardForm, { needs_intent: true } );
 
 function AddCreditCard( props ) {
 	const createAddCardToken = ( ...args ) => createCardToken( 'card_add', ...args );
 	const goToBillingHistory = () => page( billingHistory );
 	const recordFormSubmitEvent = () =>
 		analytics.tracks.recordEvent( 'calypso_add_credit_card_form_submit' );
+	const shouldUseStripe = true; // TODO: how should we derive this?
```

Comment:
When I force this to `false` and use the form, I get a `createCardToken is not a function` notice in the UI, and no action is taken.

Reply:
Wow, thanks for seeing that! 😰 I mixed up my arguments in `makeAsyncCreateCardToken`